gallery/gallery/tool/codeviewer_cli/main.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Flutter team. All rights reserved.
```

Comment:
nit: 2020? Or is this a modified version of the previous `main.dart`?

Reply:
It is a modified version of the previous main.dart!